content/doc/administration/requirements/linux.adoc

```diff
@@ -0,0 +1,78 @@
+---
+layout: documentation
+title:  Linux Support Policy
+---
+
+This page documents the Linux support policy for the Jenkins controller and agents.
+
+== Scope
+
+Jenkins plugins may set additional requirements to Linux versions on controllers and/or agents.
+This page does not document such requirements.
+Please refer to plugin documentation.
+
+== Why?
+
+Theoretically, Jenkins can run everywhere where you can run a supported Java version,
+but there are some limitations in practice.
+Jenkins core and some plugins include native code or depend on Linux API and subsystems,
+and hence they rely on specific Linux versions.
+Jenkins platform specific installation packages rely on specific Linux versions.
+
+== Support levels
+
+We define multiple support levels for Linux platforms.
+
+[width="100%",cols="20%,35%,45%",options="header",]
+|===
+|Support level |Description |Platforms
+
+| **Level 1** - Full support
+| We run automated testing for these platforms, and we intend to fix the reported issues timely.
+a|
+  * 64-bit (amd64) Linux versions supported by Debian, Red Hat, SUSE, and Ubuntu
```

Comment:
Do we really run automated tests for all these platforms, I do not think so

Reply:
Good point.  We don't run automated tests on all those platforms.  We run automated tests of Jenkins core builds on Ubuntu and Windows Server 2019.  We run automated tests of weekly installers on Debian and CentOS.  We run automated tests of LTS installers on Debian and CentOS.

Should this draft and the [Windows Support Policy](https://www.jenkins.io/doc/administration/requirements/windows/) be modified to more precisely state what we test with automation?

Alternately, the statements about automated tests could be removed, since the users probably care more that we are actively maintaining certain platforms rather than the specific testing methods we use.

Reply:
Thanks to the work from @basil in https://github.com/jenkinsci/packaging, we now have [package installation tests](https://ci.jenkins.io/job/Packaging/job/packaging/job/master/) that check:

* almalinux-8
* amazonlinux-2
* centos-stream-8
* debian-10
* debian-11
* fedora-34
* fedora-35
* opensuse-leap-15
* oraclelinux-8
* rockylinux-8
* ubuntu-18-04
* ubuntu-20-04
* ubuntu-21-04
* ubuntu-21-10
* ubuntu-22-04